eng/scripts/Language-Settings.ps1

```diff
@@ -216,3 +216,31 @@ function Update-java-CIConfig($pkgs, $ciRepo, $locationInDocRepo, $monikerId=$nu
 
   Set-Content -Path $pkgJsonLoc -Value $jsonContent
 }
+
+function GetExistingPackageVersions ($PackageName, $GroupId=$null) {
+  try {
+    $Uri = 'https://search.maven.org/solrsearch/select?q=g:"' + $GroupId + '"+AND+a:"' + $PackageName +'"&core=gav&rows=20&wt=json'
+    $existingVersion = Invoke-RestMethod -Method GET -Uri $Uri
+    return $existingVersion.response.docs.v
+  }
+  catch {
+    LogError "Failed to retieve package versions. `n$_"
+    return $null
+  }
+}
+
+function SetPackageVersion ($PackageName, $Version, $ServiceDirectory, $ReleaseDate, $BuildType, $GroupId) {
+  if($null -eq $ReleaseDate)
+  {
+    $ReleaseDate = Get-Date -Format "yyyy-MM-dd"
+  }
+  python "$EngDir/versioning/set_versions.py" --build-type $BuildType --new-version $Version --ai $PackageName --gi $GroupId
```

Comment:
Do we need to install any python dependencies to call these scripts?

Reply:
Aside from python, no other dependency is required.